recipes/bamhash/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: bamhash
+  version: 1.0
+
+source:
+  git_url: https://github.com/DecodeGenetics/BamHash.git
+  git_tag: v1.0
+
+requirements:
+  build:
+    - gcc # [linux]
+    - llvm # [osx]
+    - openssl
+    - zlib
+
+  run:
+    - openssl
+    - libgcc
```

Comment:
This is only needed for linux, if it's needed at all.
The zlib error look wired :(
I need to look at this closer if I find time.